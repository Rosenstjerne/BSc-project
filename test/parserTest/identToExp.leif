var int a,b,c;
a = 5;
b = a;
c = a + b;

print(c); // 10