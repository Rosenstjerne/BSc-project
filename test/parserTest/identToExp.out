function null main() {
	var int a, b, c;
	
	a = 5;
	b = a;
	c = (a + b);
	print(c);
	
}
